tmt/steps/execute/sti.py

```diff
@@ -0,0 +1,108 @@
+import os
+import click
+import tmt
+
+from tmt.utils import GeneralError
+
+# The default playbook used to run the tests
+DEFAULT_PLAYBOOK = 'tests/tests.yml'
+
+# Ansible inventory file required to run the tests
+ANSIBLE_INVENTORY = """
+[localhost]
+sut  ansible_host={hostname} ansible_user=root ansible_remote_tmp={remote_tmp}
+"""
+
+
+class ExecutorSTI(tmt.steps.execute.ExecutePlugin):
```

Comment:
Unit tests are failing. It would be also nice to include at least a simple `tier: 3` test.

Reply:
will add some, keeping open for now